src/CommonITILObject.php

```diff
@@ -7250,6 +7250,34 @@ class_exists($validation_class) && $params['with_validations']
             }
         }
 
+        // add autobumps to timeline
+        $autobump_obj = new ITILAutoBump();
+        $autobumps = $autobump_obj->find(['items_id'  => $this->getID()]);
+        foreach ($autobumps as $autobump_id => $autobump) {
+            $autobump_obj = ITILAutoBump::getByID($autobump_id);
+            $pending_reason = $autobump_obj->getPendingReason();
+            $followup_template = ITILFollowupTemplate::getById($pending_reason->fields['itilfollowuptemplates_id']);
+            $content = sprintf(
+                '<span>%1$s%2$s (<span data-bs-toggle="popover" data-bs-html="true" data-bs-sanitize="true" data-bs-content="%3$s"><u>%4$s</u></span>)</span>',
+                '<i class="ti ti-refresh-alert text-warning me-1"></i>',
+                __('Auto bump'),
+                $followup_template->fields['content'],
```

Comment:
`$followup_template` can be false.
The case is when the pending reason chosen doesn't have any followup template set, and we still choose manually (from the final followup form) an automatic followup (reminder/autobump/recall/whatever)

In this case, the following error triggers:
```
[2023-05-26 11:09:51] glpiphplog.WARNING:   *** PHP Warning (2): Attempt to read property "fields" on bool in /var/www/html/glpi/10.1.git/src/CommonITILObject.php at line 7264
  Backtrace :
  src/CommonITILObject.php:7511                      CommonITILObject->getTimelineItems()
  src/Ticket.php:912                                 CommonITILObject->addDefaultFormTab()
  src/CommonGLPI.php:336                             Ticket->defineTabs()
  src/CommonGLPI.php:934                             CommonGLPI->defineAllTabs()
  src/CommonGLPI.php:1255                            CommonGLPI->showTabsContent()
  src/CommonDBTM.php:6507                            CommonGLPI->display()
  front/ticket.form.php:259                          CommonDBTM::displayFullPageForItem()
  public/index.php:82                                require()
  
[2023-05-26 11:09:51] glpiphplog.WARNING:   *** PHP Warning (2): Trying to access array offset on value of type null in /var/www/html/glpi/10.1.git/src/CommonITILObject.php at line 7264
  Backtrace :
  src/CommonITILObject.php:7511                      CommonITILObject->getTimelineItems()
  src/Ticket.php:912                                 CommonITILObject->addDefaultFormTab()
  src/CommonGLPI.php:336                             Ticket->defineTabs()
  src/CommonGLPI.php:934                             CommonGLPI->defineAllTabs()
  src/CommonGLPI.php:1255                            CommonGLPI->showTabsContent()
  src/CommonDBTM.php:6507                            CommonGLPI->display()
  front/ticket.form.php:259                          CommonDBTM::displayFullPageForItem()
  public/index.php:82 
```